hack/quickstart/quickstart-openstack.md

```diff
@@ -0,0 +1,162 @@
+
+```
+$ openstack image list | grep -i "container linux"
++--------------------------------------+-------------------------------------------+--------+
+| ID                                   | Name                                      | Status |
++--------------------------------------+-------------------------------------------+--------+
+...
+| 90f57210-9354-4a2f-852e-d844237fbbad | Container Linux 1235.9.0                  | active |
+...
++--------------------------------------+-------------------------------------------+--------+
+```
+
+Or import one into Glance using the [CoreOS Linux OpenStack instructions](https://coreos.com/os/docs/latest/booting-on-openstack.html#import-the-image).
+
+#### Find a machine flavor
+
+Find a machine flavor and use the ID as the value for the `flavor_id` terraform variable.
+
+```
+$ openstack flavor list
++--------------------------------------+-----------------+--------+------+-----------+-------+-----------+
+| ID                                   | Name            |    RAM | Disk | Ephemeral | VCPUs | Is Public |
++--------------------------------------+-----------------+--------+------+-----------+-------+-----------+
+...
+| bbcb7eb5-5c8d-498f-9d7e-307c575d3566 | v1-standard-1   |   1024 |   40 |         0 |     2 | True      |
+...
++--------------------------------------+-----------------+--------+------+-----------+-------+-----------+
+```
+
+### Cluster size
+
+The terraform variable `controller_count` specifies the amount of the controller nodes.
+The terraform variable `worker_count` specifies the amount of worker nodes.
+
+Customize `openstack/config.tf` to add these image ids, ssh key, etc or use the terraform `-var` argument below to customize the configuration.
+
+### Launch Nodes
+
+Launch nodes using [Terraform](https://www.terraform.io/downloads.html):
+
+This relies on the OpenStack credentials in the `OS_` environment variables used by the `glance` and `openstack` tool.
+
+```
+$ terraform apply -var 'key_pair=user' -var 'image_id=90f57210-9354-4a2f-852e-d844237fbbad' -var 'flavor_id=bbcb7eb5-5c8d-498f-9d7e-307c575d3566' openstack/
+```
+
+### Bootstrap Master
+
+*Replace* `<node-ip>` with the EXTERNAL_IP from output of `openstack server list --format=json --name 'control_node_.' | jq .[].Networks`.
+
+```
+$ IDENT=~/.ssh/user-openstack ./init-master.sh <node-ip>
+```
+
+After the master bootstrap is complete, you can continue to add worker nodes. Or cluster state can be inspected via kubectl:
+
+```
+$ kubectl --kubeconfig=cluster/auth/kubeconfig get nodes
+```
+
+### Add Workers
+
+*Replace* `<node-ip>` with the EXTERNAL_IP from output of `openstack server list --format=json --name 'worker_node_.' | jq .[].Networks`.
+
+Initialize each worker node by replacing `<node-ip>` with the EXTERNAL_IP from the commands above.
+
+```
+$ IDENT=~/.ssh/user-openstack ./init-worker.sh <node-ip> cluster/auth/kubeconfig
```

Comment:
thanks for the hint, I will address this in the next iteration once the cluster is fully functioning.